### PR TITLE
first task

### PR DESCRIPTION
**Pull request by monuelDePari**

### CODE REVIEW

File: Task1_Sum_of_List.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        import random
+        for t in range(x):
+            some_list.append(random.randint(1, 100))
+            sum += some_list[-1]
+        average = sum / len(some_list)
+        print(some_list, sum, average)
+
+
+f = Sum_List()
+f.generate_random()
```
Comment (polyakovartem) [inline Task1_Sum_of_List.py]: https://docs.python.org/3/library/__main__.html

File: Task1_Sum_of_List.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Sum_List:
```
Comment (polyakovartem) [inline Task1_Sum_of_List.py]: PEP8 violation

File: Task1_Sum_of_List.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    sum = 0
+    average = 0
+
+    def generate_random(self, some_list=some_list, sum=sum, average=average):
+        x = int(input())
+        import random
+        for t in range(x):
+            some_list.append(random.randint(1, 100))
+            sum += some_list[-1]
+        average = sum / len(some_list)
```
Comment (polyakovartem) [inline Task1_Sum_of_List.py]: module random allows to squash all of these into one line

**Review by polyakovartem — CHANGES_REQUESTED** [review]

File: Task1_Sum_of_List.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
+from pip._vendor.msgpack.fallback import xrange
```
Comment (polyakovartem) [inline Task1_Sum_of_List.py:1]: why do you need this import?

**Comment by monuelDePari** [comment]
Pycharm underlined it and suggest to import so I did it

вт, 15 жовт. 2019 о 11:35 Artem <notifications@github.com> пише:

**Pull request merged**